css/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+	background-image: url(../images/background.jpg);
+	background-size: cover;
+}
+
+#header {
+	text-align: center;
+	width: 50vw;
+	margin: 0 auto 1em auto;
+}
+
+#pic1 {
+	margin-left: auto;
+	margin-right: auto;
+	max-height: 70vh;
+}
+
+#picContain {
+	height: 70vh;
+
+}
+
+/*#picContain {
```

Comment:
Try not to leave commented-out code in your files. If you need to see a previous version of your code, you can look at a previous commit. If it's not in a previous commit, commit more!